src/Sentry/SentryOptions.cs

```diff
@@ -680,7 +680,10 @@ public SentryOptions()
                     "ServiceStack"
             };
 
-            InAppInclude = Array.Empty<string>();
+            InAppInclude = new[]
+            {
+                "Sentry.Samples."
+            };
```

Comment:
It's fair to say users run our sample always in debug mode? In that case we can avoid having this ship out in production 

```suggestion
#if DEBUG
            InAppInclude = new[]
            {
                "Sentry.Samples."
            };
#endif
```

Reply:
Great idea.  I was exploring a different path with  frames from `Assembly.GetEntryAssembly()` being treated as in-app, but it's a rabbit hole due to all the ways apps can be run.  I'll just take your idea instead.  Don't want to waste too much time on this.